spec/support/sdam_monitoring/standalone.yml

```diff
@@ -0,0 +1,74 @@
+description: "Monitoring a standalone connection"
+uri: "mongodb://a:27017"
+phases:
+  -
+    responses:
+      "a:27017":
+        ok: 1
+        ismaster: true
+        minWireVersion: 0
+        maxWireVersion: 4
+    outcome:
+      events:
+        -
+          topology_opening_event:
+            topologyId: "42"
+        -
+          server_opening_event:
+            topologyId: "42"
+            address: "a:27017"
+        -
+          server_description_changed_event:
+            topologyId: "42"
+            address: "a:27017"
+            previousDescription:
+              address: "127.0.0.1:27017"
+              arbiters: []
+              electionId: null
+              error: null
+              hosts: []
+              maxWireVersion: 0
+              me: null
+              minWireVersion: 0
+              passives: []
+              primary: null
+              setName: null
+              setVersion: null
+              tags: []
+              type: "Unknown"
+            newDescription:
+              address: "a:27017"
+              arbiters: []
+              electionId: null
+              error: null
+              hosts: []
+              maxWireVersion: 4
+              me: null
+              minWireVersion: 0
+              passives: []
+              primary: null
+              setName: null
+              setVersion: null
+              tags: []
+              type: "Standalone"
+        -
+          topology_description_changed_event:
+            topologyId: "42"
+            previousDescription:
+              type: "Unknown"
+              setName: null
+              maxSetVersion: null
+              maxElectionId: null
+              servers: [ "127.0.0.1:27017" ]
```

Comment:
The SDAM spec states the default is one server of localhost:27017: https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#topologydescription